src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,9 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    Machine[] machine = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
+for (int i = 0;i < machine.length;i++) {
```

Comment:
The `for` loop is not inside any method or constructor. You need to place this loop inside a `main` method or another method to execute it properly.